src/contexts/NetworkProvider/NetworkProvider.js

```diff
@@ -74,16 +74,16 @@ const NetworkProvider = ({ value: propValue, children }) => {
           ...currentMlAddresses.mlChangeAddresses,
         ]
       : []
```

Comment:
in case if `currentMlAddresses` not defined better to add one condition before fetch, on this line
something like that:
```js
if(!currentMlAddresses) return
```

Reply:
please revert last commit